architecture/additional_concepts/authentication.adoc

```diff
@@ -161,21 +161,20 @@ To register additional clients:
 
 ----
 $ oc create -f <(echo '
-{
-  "kind": "OAuthClient",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "demo" <1>
-  },
-  "secret": "...", <2>
-  "redirectURIs": [
-    "http://www.example.com/" <3>
-  ]
-}')
+kind: OAuthClient
+apiVersion: v1
+metadata:
+ name: demo <1>
+secret: "..." <2>
+redirectURIs:
+ - "http://www.example.com/" <3>
+grantMethod: prompt <4>
+')
 ----
 <1> The `name` of the OAuth client is used as the `client_id` parameter when making requests to `_<master>_/oauth/authorize` and `_<master>_/oauth/token`.
 <2> The `secret` is used as the `client_secret` parameter when making requests to `_<master>_/oauth/token`.
 <3> The `redirect_uri` parameter specified in requests to `_<master>_/oauth/authorize` and `_<master>_/oauth/token` must be equal to (or prefixed by) one of the URIs in `redirectURIs`.
+<4> The `grantMethod` is used to determine what action to take when this client requests tokens and has not yet been granted access by the user. Uses the same values seen in link:./../../install_config/configuring_authentication.html#grant-options[Grant Options].
```

Comment:
We are now using xrefs instead of links for internal paths, so:
s/link:../../../install_config/configuring_authentication.html#grant-options[Grant Options]/xref:configuring_authentication.adoc#grant-options[Grant Options]